src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -72,6 +74,19 @@
 
           <% } %>
 
+    <p>Status: </p>
+    <p><font size="4">${status}</font></p>
+
+       <% if (statusUrl != ""){ %>
```

Comment:
Make sure you do string comparison using the `equals()` function instead of the `==` operator. This tests that they're the same exact instance, not that they contain the same content.